src/cmd/linuxkit/azure.go

```diff
@@ -307,9 +314,9 @@ func setVirtualMachineParameters(storageAccountName string, networkInterfaceID,
 				VMSize: compute.StandardDS1,
 			},
 			OsProfile: &compute.OSProfile{
-				ComputerName:  to.StringPtr("linuxkit"),
-				AdminUsername: to.StringPtr("dummyusername"),
-				AdminPassword: to.StringPtr("DummyPassword!123"),
+				ComputerName:  to.StringPtr(defaultComputerName),
+				AdminUsername: to.StringPtr(unusedAdminUsername),
+				AdminPassword: to.StringPtr(unusedPassword),
```

Comment:
Yeah, the password requirements are below (3/4 are mandatory):

Has lower characters 
Has upper characters 
Has a digit 
Has a special character (Regex match [\W_]) 

Will add a comment.